pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/RebalanceSummaryResult.java

```diff
@@ -240,6 +240,45 @@ public Map<String, ServerSegmentChangeInfo> getServerSegmentChangeInfo() {
     }
   }
 
+  public static class ConsumingSegmentSummary {
+    private final int _numConsumingSegmentsToBeMoved;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private final Integer _maxBytesConsumingSegmentsToCatchUp;
```

Comment:
Also, I thought we had discussed capturing the top 10 (or top X) segments and dumping their offset difference and time difference. here I see you've only captured the max?